packages/editor-ui/src/components/RunData.vue

```diff
@@ -1068,6 +1069,16 @@ export default mixins(
 			node() {
 				this.init();
 			},
+			inputData:{
+				handler(data: INodeExecutionData[]) {
+					if(this.paneType && data){
+						const isDataEmpty = data.length === 0 || data.every(isEmpty) || data.every(item => isEmpty(item.json));
+						this.$store.commit('ui/setNDVPanelDataIsEmpty', { panel: this.paneType, isEmpty: isDataEmpty });
```

Comment:
just so you know.. this is not perfect.. it's missing the fact that branches can be empty.. but there can still be data. it's not a major issue for this particular tooltip..

Reply:
ok, let's talk about how can this data emptiness be better determined